Panic processing files

I got this processing files:

```
panic: runtime error: slice bounds out of range

goroutine 83 [running]:
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*commentElems).Split(0xc4208dc398, 0xc42001a8b4, 0x2, 0xc4208dc398)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/semantic.go:160 +0x8eb
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*commentUAST).Check(0xc42035f3e0, 0xc421712900, 0xb97d20, 0xc421435180, 0xc420908bd8, 0xffffffffffffff01, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/semantic.go:260 +0xd3
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.Fields.CheckObj(0xc420432de0, 0x2, 0x2, 0xc421712900, 0xc421460ed0, 0xef9e70, 0xc4203bf200, 0xc4208dc7c8)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:809 +0x15f
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.Obj.CheckObj(0xc4203a5f80, 0xc421712900, 0xc421460ed0, 0x6, 0xc420908bd8, 0x1)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:269 +0x61
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*opObjJoin).CheckObj(0xc4203c0750, 0xc421712900, 0xc4210ef260, 0xc4210ef260, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:575 +0x1cf
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.checkObj(0xb972a0, 0xc4203c0750, 0xc421712900, 0xb97cc0, 0xc4210ef260, 0x532a00, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:372 +0x5c
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*opObjJoin).Check(0xc4203c0750, 0xc421712900, 0xb97cc0, 0xc4210ef260, 0xc4203c0990, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:554 +0x55
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.mappings.Do.func1(0xb97cc0, 0xc4210ef260, 0xc4208dd170, 0xb97ba0, 0xebd9c0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/transformer.go:326 +0x1d6
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc4210ef260, 0xc4208dd170, 0xb97cc0, 0xc421460a80, 0x1)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:875 +0x234
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97b40, 0xc421110680, 0xc4208dd170, 0xb97b40, 0xc4211106e0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:863 +0x12e
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc42111a420, 0xc4208dd170, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:848 +0x3a4
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc421818bd0, 0xc4208dd170, 0xc420434da0, 0xc421819900, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:848 +0x3a4
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97b40, 0xc421918e60, 0xc4208dd170, 0xb97b40, 0xc420434da0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:863 +0x12e
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc421819800, 0xc4208dd170, 0xf243d0, 0xaec8c0, 0xc421585770)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:848 +0x3a4
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.mappings.Do(0xf05480, 0x13, 0x13, 0xc4203e36b0, 0xc4203e3a70, 0x0, 0x0, 0x0, 0xb97cc0, 0xc421819800, ...)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/transformer.go:305 +0xeb
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.Transforms.do.func1(0xb09826, 0x8, 0xc4203e6320, 0x1, 0x1, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/transforms.go:99 +0x16b
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.Transforms.do(0xb07245, 0x6, 0xc42038dad0, 0x1, 0x1, 0xef2260, 0x2, 0x2, 0xc4203e6320, 0x1, ...)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/transforms.go:133 +0x4e3
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.Transforms.Do(0xb07245, 0x6, 0xc42038dad0, 0x1, 0x1, 0xef2260, 0x2, 0x2, 0xc4203e6320, 0x1, ...)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/transforms.go:89 +0x1c7
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.(*driverImpl).Parse(0xc4200ef7c0, 0xb95100, 0xc4201da420, 0xc42001a800, 0x3544, 0xc4201da450, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/impl.go:69 +0x268
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol.(*driverServer).Parse(0xc42009b840, 0xb95100, 0xc4201da3f0, 0xc42043e300, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol/driver.go:87 +0x1af
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol._Driver_Parse_Handler.func1(0xb95100, 0xc4201da3f0, 0xaa0b60, 0xc42043e300, 0xb95100, 0xc4201da3f0, 0xb99820, 0xc420178000)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol/driver.pb.go:179 +0x86
github.com/bblfsh/csharp-driver/vendor/github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc.OpenTracingServerInterceptor.func1(0xb95100, 0xc4201da1e0, 0xaa0b60, 0xc42043e300, 0xc420158740, 0xc420158760, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc/server.go:57 +0x2ba
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol._Driver_Parse_Handler(0xa2d940, 0xc42009b840, 0xb95100, 0xc4201da1e0, 0xc420430150, 0xc420159340, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol/driver.pb.go:181 +0x167
github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).processUnaryRPC(0xc420284000, 0xb98a40, 0xc42036e000, 0xc4203f6000, 0xc4201d4000, 0xef0f80, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:1026 +0x4f9
github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).handleStream(0xc420284000, 0xb98a40, 0xc42036e000, 0xc4203f6000, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:1252 +0x1318
github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc420452150, 0xc420284000, 0xb98a40, 0xc42036e000, 0xc4203f6000)
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:699 +0x9f
created by github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:697 +0xa1
time="2019-02-06T13:49:47Z" level=error msg="request processed content 13636 bytes error: rpc error: code = Unavailable desc = transport is closing" elapsed=4.164224267s language=csharp
time="2019-02-06T13:49:50Z" level=info msg="csharp-driver version: v1.0.0 (build: 2019-01-30T17:42:45Z)" id=01d31h4t0se07c0kwmzjw7fjd5 language=csharp
time="2019-02-06T13:49:50Z" level=info msg="server listening in /tmp/rpc.sock (unix)" id=01d31h4t0se07c0kwmzjw7fjd5 language=csharp
time="2019-02-06T13:49:51Z" level=info msg="new driver instance started bblfsh/csharp-driver:latest (01d31h4t0se07c0kwmzjw7fjd5)"
panic: runtime error: slice bounds out of range

goroutine 147 [running]:
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*commentElems).Split(0xc420e00398, 0xc421a34007, 0x2cf, 0xc420e00398)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/semantic.go:184 +0x8d6
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*commentUAST).Check(0xc42035f620, 0xc421f51f40, 0xb97d20, 0xc420ff7a10, 0xc42119d5e8, 0xffffffffffffff01, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/semantic.go:260 +0xd3
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.Fields.CheckObj(0xc422027020, 0x2, 0x2, 0xc421f51f40, 0xc420f238c0, 0xef9e70, 0xc4203c3e60, 0xc420e007c8)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:809 +0x15f
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.Obj.CheckObj(0xc4203c09c0, 0xc421f51f40, 0xc420f238c0, 0xb, 0xc42119d5f8, 0x1)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:269 +0x61
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*opObjJoin).CheckObj(0xc4203c1170, 0xc421f51f40, 0xc420bceab0, 0xc420bceab0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:575 +0x1cf
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.checkObj(0xb972a0, 0xc4203c1170, 0xc421f51f40, 0xb97cc0, 0xc420bceab0, 0x532a00, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:372 +0x5c
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*opObjJoin).Check(0xc4203c1170, 0xc421f51f40, 0xb97cc0, 0xc420bceab0, 0xc4203c13b0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:554 +0x55
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.mappings.Do.func1(0xb97cc0, 0xc420bceab0, 0xc420e01170, 0xb97ba0, 0xebd9c0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/transformer.go:326 +0x1d6
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc420bceab0, 0xc420e01170, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:875 +0x234
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97b40, 0xc420308740, 0xc420e01170, 0xc421a2f140, 0xc421a2f200, 0xc420e00c00)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:863 +0x12e
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc421a2e450, 0xc420e01170, 0xb97cc0, 0xc421a2f140, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:848 +0x3a4
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc421a2f0e0, 0xc420e01170, 0xc421fdf290, 0xf25200, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:848 +0x3a4
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97b40, 0xc420aa45c0, 0xc420e01170, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:863 +0x12e
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc42129c090, 0xc420e01170, 0xf243d0, 0xaec8c0, 0xc420b23d10)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:848 +0x3a4
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.mappings.Do(0xf05480, 0x13, 0x13, 0xc4203e36b0, 0xc4203e3a70, 0x0, 0x0, 0x0, 0xb97cc0, 0xc42129c090, ...)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/transformer.go:305 +0xeb
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.Transforms.do.func1(0xb09826, 0x8, 0xc4203e8320, 0x1, 0x1, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/transforms.go:99 +0x16b
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.Transforms.do(0xb07245, 0x6, 0xc42038dad0, 0x1, 0x1, 0xef2260, 0x2, 0x2, 0xc4203e8320, 0x1, ...)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/transforms.go:133 +0x4e3
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.Transforms.Do(0xb07245, 0x6, 0xc42038dad0, 0x1, 0x1, 0xef2260, 0x2, 0x2, 0xc4203e8320, 0x1, ...)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/transforms.go:89 +0x1c7
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.(*driverImpl).Parse(0xc4200ef720, 0xb95100, 0xc421a230b0, 0xc421a34000, 0xfe5, 0xc421a23110, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/impl.go:69 +0x268
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol.(*driverServer).Parse(0xc42009b840, 0xb95100, 0xc421a23050, 0xc42043d700, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol/driver.go:87 +0x1af
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol._Driver_Parse_Handler.func1(0xb95100, 0xc421a23050, 0xaa0b60, 0xc42043d700, 0xb95100, 0xc421a23050, 0xb99820, 0xc4211b43c0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol/driver.pb.go:179 +0x86
github.com/bblfsh/csharp-driver/vendor/github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc.OpenTracingServerInterceptor.func1(0xb95100, 0xc421a22ff0, 0xaa0b60, 0xc42043d700, 0xc420c488e0, 0xc420c48900, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc/server.go:57 +0x2ba
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol._Driver_Parse_Handler(0xa2d940, 0xc42009b840, 0xb95100, 0xc421a22ff0, 0xc4200f6770, 0xc420159340, 0x0, 0x0, 0xc420026140, 0x10)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol/driver.pb.go:181 +0x167
github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).processUnaryRPC(0xc420274000, 0xb98a40, 0xc4204ac200, 0xc4212b2100, 0xc4201d4000, 0xef0f80, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:1026 +0x4f9
github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).handleStream(0xc420274000, 0xb98a40, 0xc4204ac200, 0xc4212b2100, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:1252 +0x1318
github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc4204a20d0, 0xc420274000, 0xb98a40, 0xc4204ac200, 0xc4212b2100)
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:699 +0x9f
created by github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:697 +0xa1
time="2019-02-06T13:50:09Z" level=error msg="request processed content 4069 bytes error: rpc error: code = Unavailable desc = transport is closing" elapsed=496.01477ms language=csharp
time="2019-02-06T13:50:12Z" level=info msg="csharp-driver version: v1.0.0 (build: 2019-01-30T17:42:45Z)" id=01d31h5fwsrmq4eps6d0qhww83 language=csharp
time="2019-02-06T13:50:12Z" level=info msg="server listening in /tmp/rpc.sock (unix)" id=01d31h5fwsrmq4eps6d0qhww83 language=csharp
time="2019-02-06T13:50:13Z" level=info msg="new driver instance started bblfsh/csharp-driver:latest (01d31h5fwsrmq4eps6d0qhww83)"
panic: runtime error: slice bounds out of range

goroutine 80 [running]:
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*commentElems).Split(0xc420bf8398, 0xc421382007, 0x2ca, 0xc420bf8398)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/semantic.go:184 +0x8d6
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*commentUAST).Check(0xc42035f620, 0xc421731d60, 0xb97d20, 0xc421adbb70, 0xc420b7d948, 0xffffffffffffff01, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/semantic.go:260 +0xd3
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.Fields.CheckObj(0xc4203dea80, 0x2, 0x2, 0xc421731d60, 0xc420f71620, 0xef9e70, 0xc4203c3b00, 0xc420bf87c8)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:809 +0x15f
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.Obj.CheckObj(0xc4203c09c0, 0xc421731d60, 0xc420f71620, 0xb, 0xc420b7d958, 0x1)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:269 +0x61
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*opObjJoin).CheckObj(0xc4203c1170, 0xc421731d60, 0xc421b6f350, 0xc421b6f350, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:575 +0x1cf
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.checkObj(0xb972a0, 0xc4203c1170, 0xc421731d60, 0xb97cc0, 0xc421b6f350, 0x532a00, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:372 +0x5c
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.(*opObjJoin).Check(0xc4203c1170, 0xc421731d60, 0xb97cc0, 0xc421b6f350, 0xc4203c13b0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/ops.go:554 +0x55
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.mappings.Do.func1(0xb97cc0, 0xc421b6f350, 0xc420bf9170, 0xb97d20, 0xc4210f71c0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/transformer.go:326 +0x1d6
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc421b6f350, 0xc420bf9170, 0xc420afac70, 0xc421418700, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:875 +0x234
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97b40, 0xc4212cd300, 0xc420bf9170, 0xb97d20, 0xc420afac70, 0xc420bf8c00)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:863 +0x12e
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc421418630, 0xc420bf9170, 0xb97cc0, 0xc42197be90, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:848 +0x3a4
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc42197bdd0, 0xc420bf9170, 0xc4212afc30, 0xf25200, 0xc420bf8f00)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:848 +0x3a4
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97b40, 0xc42178c760, 0xc420bf9170, 0xb97d20, 0xc4212afc30, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:863 +0x12e
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes.Apply(0xb97cc0, 0xc42173ac60, 0xc420bf9170, 0xf243d0, 0xaec8c0, 0xc421329d10)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/nodes/node.go:848 +0x3a4
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer.mappings.Do(0xf05480, 0x13, 0x13, 0xc4203e36b0, 0xc4203e3a70, 0x0, 0x0, 0x0, 0xb97cc0, 0xc42173ac60, ...)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/uast/transformer/transformer.go:305 +0xeb
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.Transforms.do.func1(0xb09826, 0x8, 0xc4203e6320, 0x1, 0x1, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/transforms.go:99 +0x16b
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.Transforms.do(0xb07245, 0x6, 0xc42038dad0, 0x1, 0x1, 0xef2260, 0x2, 0x2, 0xc4203e6320, 0x1, ...)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/transforms.go:133 +0x4e3
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.Transforms.Do(0xb07245, 0x6, 0xc42038dad0, 0x1, 0x1, 0xef2260, 0x2, 0x2, 0xc4203e6320, 0x1, ...)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/transforms.go:89 +0x1c7
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver.(*driverImpl).Parse(0xc4200ef9a0, 0xb95100, 0xc420c8fe60, 0xc421382000, 0x1d61, 0xc420c8fe90, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/driver/impl.go:69 +0x268
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol.(*driverServer).Parse(0xc420452350, 0xb95100, 0xc420c8fe30, 0xc421330400, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol/driver.go:87 +0x1af
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol._Driver_Parse_Handler.func1(0xb95100, 0xc420c8fe30, 0xaa0b60, 0xc421330400, 0xb95100, 0xc420c8fe30, 0xb99820, 0xc4211e4870)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol/driver.pb.go:179 +0x86
github.com/bblfsh/csharp-driver/vendor/github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc.OpenTracingServerInterceptor.func1(0xb95100, 0xc420c8fdd0, 0xaa0b60, 0xc421330400, 0xc42130b5a0, 0xc42130b5c0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc/server.go:57 +0x2ba
github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol._Driver_Parse_Handler(0xa2d940, 0xc420452350, 0xb95100, 0xc420c8fdd0, 0xc4201bba40, 0xc4204466a0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/gopkg.in/bblfsh/sdk.v2/protocol/driver.pb.go:181 +0x167
github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).processUnaryRPC(0xc420274000, 0xb98a40, 0xc42036e000, 0xc4210e6400, 0xc42045ae70, 0xef0f80, 0x0, 0x0, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:1026 +0x4f9
github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).handleStream(0xc420274000, 0xb98a40, 0xc42036e000, 0xc4210e6400, 0x0)
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:1252 +0x1318
github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc420026110, 0xc420274000, 0xb98a40, 0xc42036e000, 0xc4210e6400)
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:699 +0x9f
created by github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
	/go/src/github.com/bblfsh/csharp-driver/vendor/google.golang.org/grpc/server.go:697 +0xa1
```

Sadly, I cannot provide the actual offending files.

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -365,8 +365,8 @@
     "uast/viewer",
     "uast/yaml"
   ]
-  revision = "42e2023d304c66a242444d467c48f6913748e411"
-  version = "v2.13.3"
+  revision = "d506b5408b0afda31d07e9a90343769efbfef837"
+  version = "v2.13.4"
 
 [[projects]]
   name = "gopkg.in/src-d/go-errors.v1"

